A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AltimeterIndicator.js

```diff
@@ -188,7 +188,7 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
                     line.SVGText1.setAttribute("alignment-baseline", "central");
                     line.SVGText2 = document.createElementNS(Avionics.SVG.NS, "text");
                     line.SVGText2.setAttribute("x", "-28");
-                    line.SVGText2.setAttribute("fill", "white");
+                    line.SVGText2.setAttribute("fill", "white");//blue
```

Comment:
What does this comment mean ?

Reply:
Sorry, These were original values I kept in the testing phase, thought i had removed all of these